gensrc/proto/olap_file.proto

```diff
@@ -175,6 +175,48 @@ message OLAPHeaderMessage {
     optional uint64 shard = 22;
 }
 
+enum AlterTabletStatePB {
+    RUNNING = 0;
+    FINISHED = 1;
+    FAILED = 2;
+}
+
+enum AlterTabletTypePB {
+    SCHEMA_CHANGE = 0;
+    ROLLUP = 1;
+}
+
+message AlterTabletPB {
+    required int64 related_tablet_id = 1;
```

Comment:
PB or Pb ?

Reply:
Abbreviated form may be better to written in capital.